github/REVIEWING.md

```diff
@@ -0,0 +1,341 @@
+   git remote show michpappas
+   ```
+
+1. Check the remote branch that was used to create the pull request:
+
+   ```console
+   git branch -a
+   ```
+
+   In the output of the above command you should see a line `remotes/<username>/<branch>`.
+   If you don't see that, you did something wrong and you need to remove and recreate the remote, via the `git remote` command.
+
+1. Fetch the contribution from the remote end:
+
+   ```console
+   git fetch <username>
+   ```
+
+   E.g.:
+
+   ```console
+   git fetch cluosh
+   git fetch michpappas
+   ```
+
+1. Create a local branch of the contribution:
+
+   ```console
+   git checkout -b <local-branch-name> <username>/<branch>
+   ```
+
+   where:
+
+   - `<username>` is the name of the remote
+   - `<branch>` is the name of the remote branch
+   - `<local-branch-name>` is the name of the local branch you will use
+
+   The name of the local branch is something you choose.
+   Generally, it is advised to keep the same name as `<branch>`.
+   However, it may be the case that the contributor uses a value for `<branch>` that conflicts with an existing local one (such as `main` or `master` or `testing`).
+   In that case, it is suggested to use `<username>-<branch>` for the local branch name.
+
+   E.g.:
+
+   ```console
+   git checkout -b cluosh-staging cluosh/staging
+   git checkout -b michpappas/feature/libukmemtag michpappas/michpappas/feature/libukmemtag
+   ```
+
+   In case of the first command, we used `<username>-<branch>` for the local branch.
+   This is because the remote branch name (`staging`) conflicted with a local branch name called the same (`staging`).
+
+   In the case of the second command we used the same name for the local branch as for the remote branch: `michpappas/feature/libukmemtag`.
+   Note that there is a recommendation to use, as branch name, the format `<username>/<branch-detail>`, to be able to pin the branch name to the user creating it.
+
+So, at this point you should have:
+
+- Your browser pointing out to the pull request interface on GitHub.
+- Your local repository pointing to the contents of the remote branch used in the pull request.
+
+We will detail the 5 review action below, mentioning whether they will happen either on the GitHub pull request interface or locally, on the clone of the repository.
+
+## Review Commit Metadata
+
+Reviewing the commit metadata happens locally.
+Use the command to browse through the metadata of commits:
+
+```console
+git log
+```
+
+Check the commits in the contributions and make sure they follow [good commit practices](https://cbea.ms/git-commit/).
+Generally, this means:
+
+1. The first line of the commit message (the title) is a representative line, that's not very long (ideally, but not strictly, limited at 50 characters).
+   The title line typically has a prefix that points to the type of contribution and/or the location in the repository.
+   This is generally stated as part of the repository contributor conventions.
```

Comment:
Other repos use `[feat]`/`[nfc]`/`[bug]`/`[fix]` prefixes instead of the location. It is better to consult the repository's contribution guidelines to make sure.